### PR TITLE
hide identity service behind a license field

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

File: kotsadm/pkg/handlers/app.go [hunk 1]
```diff
@@ -149,6 +150,11 @@ func responseAppFromApp(a *apptypes.App) (*ResponseApp, error) {
 		return nil, errors.Wrap(err, "failed to check if gitops is supported")
 	}
 
+	isIdentityServiceSupported, err := store.GetStore().IsIdentityServiceSupportedForVersion(a.ID, a.CurrentSequence)
```
Comment (emosbaugh) [inline kotsadm/pkg/handlers/app.go:153]: these functions sure get the app config a whole lot unnecessarily. not a fix for this pr though

**Review by emosbaugh — APPROVED** [review]

File: kotsadm/pkg/handlers/app.go (hunk 1 above)
Comment (salahalsaleh) [reply kotsadm/pkg/handlers/app.go:153]: totally agree, we should fix it

**Review by emosbaugh — APPROVED** [review]

File: kotsadm/web/src/components/apps/AppLicense.jsx [hunk 2]
```diff
@@ -204,6 +204,7 @@ class AppLicense extends Component {
                   </div>}
                 <div className="flexWrap--wrap flex alignItems--center u-marginTop--10">
                   {appLicense?.isAirgapSupported ? <span className="flex alignItems--center u-fontWeight--medium u-fontSize--small u-lineHeight--normal u-color--tundora u-marginRight--10"><span className="icon licenseAirgapIcon" /> Airgap enabled </span> : null}
+                  {/* TODO: icon */}{/* {appLicense?.isIdentityServiceSupported ? <span className="flex alignItems--center u-fontWeight--medium u-fontSize--small u-lineHeight--normal u-color--tundora u-marginLeft--5 u-marginRight--10"><span className="icon licenseVeleroIcon" /> Identity Service enabled </span> : null} */}
```
Comment (GraysonNull) [inline kotsadm/web/src/components/apps/AppLicense.jsx:207]: Why does this need to be commented out?

Comment (salahalsaleh) [reply kotsadm/web/src/components/apps/AppLicense.jsx:207]: cuz there's no icon for it

**Pull request merged**